packages/plugin-lighthouse/src/lib/runner/utils.ts

```diff
@@ -38,43 +39,46 @@ export class LighthouseAuditParsingError extends Error {
   }
 }
 
+function formatBaseAuditOutput(lhrAudit: Result): AuditOutput {
+  const { id: slug, score, numericValue, displayValue } = lhrAudit;
+  return {
+    slug,
+    score: score ?? 1, // score can be null
+    value: numericValue ?? score ?? 0, // not every audit has a numericValue
```

Comment:
❓ I copied the comments from the existing implementation and now wonder if they are superfluous. One can easily see the `score` and `numericValue` typing, and the comments don't add more details to that.

Reply:
Agree! you can remove them